client/src/pages/Login.jsx

```diff
@@ -136,141 +101,136 @@ export default function Login() {
-											shrink: true
-										}}
-										InputProps={{ classes: { input: classes.inputs } }}
-										name="email"
-										autoComplete="email"
-										autoFocus
-										helperText={touched.email ? errors.email : ''}
-										error={touched.email && Boolean(errors.email)}
-										value={values.email}
-										onChange={handleChange}
-									/>
-									<TextField
-										id="password"
-										label={<Typography className={classes.label}>Password</Typography>}
-										fullWidth
-										margin="normal"
-										InputLabelProps={{
-											shrink: true
-										}}
-										InputProps={{
-											classes: { input: classes.inputs },
-											endAdornment: <Typography className={classes.forgot}>Forgot?</Typography>
-										}}
-										type="password"
-										autoComplete="current-password"
-										helperText={touched.password ? errors.password : ''}
-										error={touched.password && Boolean(errors.password)}
-										value={values.password}
-										onChange={handleChange}
-										type="password"
-									/>
+									fullWidth
+									margin="normal"
+									InputLabelProps={{
+										shrink: true
+									}}
+									InputProps={{
+										classes: { input: classes.inputs },
+										endAdornment: <Typography className={classes.forgot}>Forgot?</Typography>
+									}}
+									type="password"
+									autoComplete="current-password"
+									helperText={touched.password ? errors.password : ''}
+									error={touched.password && Boolean(errors.password)}
+									value={values.password}
+									onChange={handleChange}
+									type="password"
+								/>
 
-									<Box textAlign="center">
-										<Button
-											type="submit"
-											size="large"
-											variant="contained"
-											color="primary"
-											className={classes.submit}
-										>
-											Login
-										</Button>
-									</Box>
+								<Box textAlign="center">
+									<Button
+										type="submit"
+										size="large"
+										variant="contained"
+										color="primary"
+										className={classes.submit}
+									>
+										Login
+									</Button>
+								</Box>
 
-									<div style={{ height: 95 }} />
-								</form>
-							)}
-						</Formik>
-					</Box>
-					<Box p={1} alignSelf="center" />
+								<div style={{ height: 95 }} />
+							</form>
+						)}
+					</Formik>
 				</Box>
-				<Snackbar
-					anchorOrigin={{
-						vertical: 'bottom',
-						horizontal: 'center'
-					}}
-					open={open}
-					autoHideDuration={6000}
-					onClose={handleClose}
-					message="Login failed"
-					action={
-						<React.Fragment>
-							<IconButton size="small" aria-label="close" color="inherit" onClick={handleClose}>
-								<CloseIcon fontSize="small" />
-							</IconButton>
-						</React.Fragment>
-					}
-				/>
-			</Grid>
-		</Grid>
+				<Box p={1} alignSelf="center" />
+			</Box>
+			<Snackbar
```

Comment:
Nitpick - these smaller components that you reuse (like the Snackbar, and even how you did the TextField, could also be reusable components used. This is a nitpick, so not a huge deal

Reply:
This would be great to add. I will make a note of this and come back to it.